src/styles/core/_vars.scss

```
@@ -81,3 +81,11 @@ $on-this-page-aside-width: 192px;
 
 // Suggest Lang
 $suggest-lang-height: 52px;
+
+// Localized Quotes
+$localizedQuotes: (
+  ja-JP: "「" "」",
+);
+
+// Localized Fonts
+$localizedFonts: ()
```

I would make them default + add some comments how to use them. Maybe the structure for the `$localizedFonts`

```suggestion
$localizedQuotes: (
  ja-JP: "「" "」",
) !default;

// Localized Fonts
$localizedFonts: () !default;
```

---

https://github.com/apple/swift-docc-render/pull/677/commits/90f0b4ad049fd9087e1cc115574c02c1fd3ac7d9